api/RESTful API/Posts/postRouter.js

```diff
@@ -0,0 +1,55 @@
+const express = require('express');
+const router = express.Router();
+const userDB = require('../../../data/helpers/postDb');
+
+//////////////POST SECTION///////////////
+
+// get all posts
+router.get('/api/posts/all', (req, res) => {
+	postDB
+		.get()
+		.then((post) => {
+			res.status(200).json(post);
+		})
+		.catch((err) => res.status(500).json({ message: "Can't retrieve post data!!" }));
+	userDB;
+});
+// test success
+
+// get post by id
+router.get('/api/posts/:id', (req, res) => {
```

Comment:
The path to posts is really complicated. You are defining `api/post` as a parent route in the server bu tthen You slap another route on top of that so the end result is `api/post/api/posts/:id` which is long and complicated but it seems to be a typo because `user` endpoints look correct.